pkg/shares/share_splitting.go

```diff
@@ -63,9 +63,10 @@ func Split(data coretypes.Data, useShareIndexes bool) ([]Share, error) {
 	currentShareCount += len(blobShares)
 	tailShares := TailPaddingShares(wantShareCount - currentShareCount)
 	shares := make([]Share, 0, data.SquareSize*data.SquareSize)
-	shares = append(append(append(append(
+	shares = append(append(append(append(append(
```

Comment:
note to self that we should create a generic helper like `appendAll` or something to get rid of alllllll the giant appends we have

Reply:
Good idea or alternatively:

```go
	shares := make([]Share, 0, data.SquareSize*data.SquareSize)
	shares = append(shares, txShares...)
	shares = append(shares, pfbTxShares...)
	shares = append(shares, padding...)
	shares = append(shares, blobShares...)
	shares = append(shares, tailShares...)
```

I don't have a strong preference. Part of me thinks that since Go doesn't have a built-in `appendAll` it may be more idiomatic to use multiple `append()` invocations

Reply:
sure that works too